cardano-api/src/Cardano/Api/Address.hs

```diff
@@ -389,6 +389,10 @@ anyAddressInEra (AddressShelley addr) =
       LegacyByronEra      -> Nothing
       ShelleyBasedEra era -> Just (AddressInEra (ShelleyAddressInEra era) addr)
 
+toAddressAny :: Address addr -> AddressAny
+toAddressAny = \case
+  a@ShelleyAddress{} -> AddressShelley a
+  a@ByronAddress{}   -> AddressByron a
```

Comment:
local style would be
```
toAddressAny a@ShelleyAddress{} = AddressShelley a
toAddressAny a@ByronAddress{}   = AddressByron a
```
no need for case or lambda case at all.

Reply:
This needs to be exported, both from this module and from `Cardano.API` and `Cardano.Api.Typed`

Reply:
Done.